platform/linux-generic/include/odp/api/plat/hash_inlines.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef ODP_PLAT_HASH_INLINES_H_
+#define ODP_PLAT_HASH_INLINES_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <odp/api/abi/hash_crc32.h>
+
+#include <stdint.h>
+
+/** @cond _ODP_HIDE_FROM_DOXYGEN_ */
+
+#ifndef _ODP_NO_INLINE
+	/* Inline functions by default */
+	#define _ODP_INLINE static inline
```

Comment:
Hi @matiaselo, can you please look into this error generated by checkpatch? Thank you!

Reply:
Hi @Malvika-Gupta, this is a false positive and can be ignored. We have this same code line in all *_inlines.h files.